libs/wire-api/src/Wire/API/Routes/Internal/Galley.hs

```diff
@@ -57,8 +56,7 @@ import Wire.API.Team.SearchVisibility
 import Wire.API.User.Client
 
 type LegalHoldFeaturesStatusChangeFederatedCalls =
```

Comment:
Why is this type still necessary?

Reply:
It's not. Just a left over... will remove...